scripts/ConvertCSVtoDat.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
Add description of what the script is supposed to do

Reply:
Few lines of description are added.